dataland-keycloak/start_keycloak.sh

```diff
@@ -1,14 +1,29 @@
 #!/bin/bash
 set -eu
 
+mode=$1
+dataland_realm_folder=/opt/keycloak/datalandsecurity
 script_dir="$(dirname "$0")"
 cd $script_dir
 
-if [[ "$1" == initialize ]]; then
+if [[ "$mode" == initialize ]]; then
   echo "Initializing new keycloak realms"
- ./kc.sh import --file /opt/keycloak/data/import/master-realm.json
- ./kc.sh start --import-realm
+  mkdir -p $dataland_realm_folder
+  cp /keycloak_users/datalandsecurity-users-*.json $dataland_realm_folder || echo "No importable users exist"
+  rm $(grep -E -l '"username" : "data_(reader|uploader)"' $dataland_realm_folder/datalandsecurity-users-*.json) || echo "No technical users to be cleaned up"
+  rm $(grep -E -l '"username" : "test_user.*@dataland.com"' $dataland_realm_folder/datalandsecurity-users-*.json) || echo "No test users to be cleaned up"
+  cp /keycloak_realms/datalandsecurity-realm.json $dataland_realm_folder
```

Comment:
Is this old information anyway overwritten with the restart or why is it not removed?

Reply:
The only container having access to this data keycloak-initializer which gets shut down in initialize_keycloak.sh. The corresponding folder on the server is cleaned up in start_and_deploy_to_server.sh